core/src/test/scala/unit/kafka/server/DynamicBrokerConfigTest.scala

```diff
@@ -450,54 +450,52 @@ class DynamicBrokerConfigTest {
   }
 
   @Test
-  def testPasswordConfigEncryption(): Unit = {
-    val props = TestUtils.createBrokerConfig(0, TestUtils.MockZkConnect, port = 8181)
+  def testPasswordConfigNotEncryption(): Unit = {
+    val props = TestUtils.createBrokerConfig(0, null, port = 8181)
```

Comment:
It looks like following test cases still use `MockZkConnect`. Could you also update it? Thanks.

* testDynamicConfigInitializationWithoutConfigsInZK
* testUpdateRemoteLogManagerDynamicThreadPool
* testRemoteLogDynamicThreadPoolWithInvalidValues